src/utilities/plugin-utils.ts

```diff
@@ -185,7 +185,6 @@ export const getGlossaryEmbeddable = (activity: Activity) => {
     : undefined;
   return embeddablePlugin;
 };
-function addUsedPlugin(plugin: Plugin) {
-  throw new Error("Function not implemented.");
-}
-
+// function addUsedPlugin(plugin: Plugin) {
+//   throw new Error("Function not implemented.");
+// }
```

Comment:
Is this supposed to be commented out? I see it's not being used in line 54 anymore. Is it no longer used at all? If not, should it just be removed?